core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -191,6 +191,12 @@ private CarbonCommonConstants() {
 
   public static final String LOCK_PATH_DEFAULT = "";
 
+  /**
+   * Specifies the lock implement class.
```

Comment:
please mention what are the options for this property

Reply:
done, the class should be the implement of ICarbonLock